### ISSUE TITLE
Add development guides

### ISSUE THREAD

**Issue by FooBarWidget** [phusion/passenger_library#16]
Besides the quickstart and walkthroughs, all the documentation is oriented towards production. It may be a good idea to add some development guides too. A user told us that he was [confused by the lack of development guides](https://groups.google.com/d/msg/phusion-passenger/TLYS8HZ_rr4/-Mnw2S-iDQAJ). He wants to develop multiple projects at once, but does not know how to set that up on a development machine where e.g. security with user accounts are not necessary.

We should add a development guide section in which we explain:

 * How to set up multiple apps for development
 * How code reloading works (copy-paste from basics walkthrough)

**Comment by daBee** [comment]
Perhaps a suite of files for two vhosts and associated nginx setup.  Also, how to sniff out the proper passenger-root by using passenger-config --root to make sure everything is in line.

**Comment by FooBarWidget** [comment]
Fixing passenger_root with passenger-config --root is already documented: https://www.phusionpassenger.com/library/config/nginx/reference/#how-to-fix-passenger_root

**Comment by daBee** [comment]
Yes, but having users jump around documentation for a workflow is where people get their instructions broken up and confusing.  As documented as it was, I didn't find it and went with what the installation of Passenger told me to use, which was the incorrect entry.

**Comment by FooBarWidget** [comment]
All right. But why do you think it belongs to the development guides? Why not somewhere else, e.g. under troubleshooting?

**Comment by daBee** [comment]
If you lead noobs down this path with a checklist, then they won't need the troubleshooting.  Happy first customer is best.
